File: src/main/scala/org/xbery/artbeams/articles/repository/ArticleCategoryFilter.scala
package org.xbery.artbeams.articles.repository

/**
  * @author Radek Beran
  */
case class ArticleCategoryFilter(articleId: Option[String], categoryId: Option[String])

object ArticleCategoryFilter {
  lazy val Empty = ArticleCategoryFilter(None, None)
}

File: src/main/kotlin/org/xbery/artbeams/articles/repository/ArticleCategoryFilter.kt
package org.xbery.artbeams.articles.repository

/**
 * @author Radek Beran
 */
data class ArticleCategoryFilter(val articleId: String?, val categoryId: String?) {
    companion object {
        val Empty: ArticleCategoryFilter = ArticleCategoryFilter(null, null)
    }
}
